packages/canon/src/components/Container/Docs.mdx

```diff
@@ -0,0 +1,74 @@
+import { Meta, Unstyled, Source } from '@storybook/blocks';
+import * as ContainerStories from './Container.stories';
+import { Title, Text } from '../../../docs/components';
+import { PropsTable, getBoxProps } from '../../../docs/components/PropsTable';
+import { spacingProperties } from '../../layout/sprinkles.css';
+import { containerProperties } from './sprinkles.css';
+
+<Meta of={ContainerStories} />
+
+<Unstyled>
+
+<Title type="h1">Container</Title>
+
+<Text>
+  The container component let you use our default max-width and center the
+  content on the page.
+</Text>
+
+<Title type="h2">Installation</Title>
+<Source code={`yarn install @backstage/canon/container`} language="bash" dark />
```

Comment:
Hold up, this doesn't actually work 😅 

Needs to be an import or install of `@backstage/canon`

Reply:
Oh you're right. Radix is actually creating dedicated packages for each of their component. Not sure what I was thinking here :)

Reply:
I fixed that.